ibis/expr/api.py

```diff
@@ -1234,23 +1233,29 @@ def _generic_summary(arg, exact_nunique=False, prefix=None):
       Compute the exact number of distinct values (slower)
     prefix : string, default None
       String prefix for metric names
+    suffix : string, default None
+      String suffix for metric names
 
     Returns
     -------
     summary : (count, # nulls, nunique)
     """
-    metrics = [arg.count(), arg.isnull().sum().name('nulls')]
-
     if exact_nunique:
         unique_metric = arg.nunique().name('uniques')
     else:
         unique_metric = arg.approx_nunique().name('uniques')
 
-    metrics.append(unique_metric)
-    return _wrap_summary_metrics(metrics, prefix)
+    metrics = [arg.count(), arg.isnull().sum().name('nulls'), unique_metric]
+
+    if prefix:
+        metrics = [m.name(f"{prefix}{m.get_name()}") for m in metrics]
```

Comment:
You could probably do this in one line with:

```python
[m.name(f"{prefix or ''}{m.get_name()}{suffix or ''}") for m in metrics]
```

Reply:
Silly me :) Updating.